src/functions/RepresentableFunction.h

```diff
@@ -38,20 +38,21 @@
 #include "MRCPP/constants.h"
 #include "MRCPP/mrcpp_declarations.h"
 #include "trees/NodeIndex.h"
+#include "MRCPP/utils/math_utils.h"
 
 namespace mrcpp {
 
-template <int D> class RepresentableFunction {
+template <int D, typename T> class RepresentableFunction {
```

Comment:
Why not have a default for `T` here?
```suggestion
template <int D, typename T = double> class RepresentableFunction {
```

Reply:
it is already set as default in mrcpp_declarations.h